library/Icingadb/Widget/ItemList/StateListItem.php

```diff
@@ -60,6 +63,11 @@ protected function assembleIconImage(BaseHtmlElement $iconImage)
     {
         if (isset($this->item->icon_image->icon_image)) {
             $src = $this->item->icon_image->icon_image;
+
+            if (strpos($src, '/') === false) {
+                $src = (string) Url::fromPath(static::DEFAULT_ICON_PATH . $src);
```

Comment:
Don't use Url::fromPath, it doesn't have to be an absolute path, that's done by the browser.